zulipterminal/config/keys.py

```diff
@@ -370,6 +370,11 @@ class KeyBinding(TypedDict):
         'help_text': 'Cut backwards to the start of the current word',
         'key_category': 'msg_compose',
     },
+    'CUT_WHOLE_LINE': {
+        'keys': ['meta x'],
+        'help_text': 'Cut the current line',
+        'key_category': 'msg_compose',
+    },
```

Comment:
We might end up squashing this and the previous commit, since they're both about missing documented keys, but this is clearer as a separate commit at this point :+1:

Unless I'm missing something, the title suggests that we're adding a command, not documenting an existing command.